src/full/Agda/Syntax/Abstract.hs

```diff
@@ -145,20 +145,36 @@ type RecordAssigns = [RecordAssign]
 -- | Renaming (generic).
 type Ren a = Map a (List1 a)
 
-data ScopeCopyInfo = ScopeCopyInfo
-  { renModules :: Ren ModuleName
-  , renNames   :: Ren QName }
-  deriving (Eq, Show, Generic)
+-- | Size of the range of the renaming.
+renamingSize :: Ren a -> Int
+renamingSize = Map.foldl' (\n a -> n + length a) 0
```

Comment:
The use of `a` which is also the name of the type parameter confuses me.  Maybe `xs`?